classification2.Rmd

```diff
@@ -925,7 +962,7 @@ In particular, you
 2. tune each one using cross validation
 3. pick the subset of predictors that gives you the highest cross-validation accuracy  
 
-Best subset selection is applicable to any  classification method (KNN or otherwise).
+Best subset selection is applicable to any classification method ($K$-NN or otherwise).
```

Comment:
full name? forget if we are doing full name consistently throughout

Reply:
I think we have $K$-NN in a couple of places

Reply:
OK. I don't feel strongly, so happy to keep it